lessons/module-3/react-1-the-what-and-the-why.md

```diff
@@ -49,19 +48,22 @@ Use the following prompts to help guide your discussion.
 People will define React in many different ways, but at its core, React is:
 
 ```
-A client-side JavaScript framework that allows you to easily and efficiently manipulate the DOM based on application data and how it changes in response to user interaction.
+A client-side JavaScript library that allows you to easily and efficiently manipulate the DOM based on application data and how it changes in response to user interaction.
 ```
 
 Let's break this definition down a little bit.
 
-### What is a framework?
+### Library vs Framework?
 
-The term framework is often used interchangeably with the term 'library', but they're actually a bit different. Libraries are usually a little bit smaller, and generally serve the purpose of providing us with abstractions over complex code that we would otherwise have to write ourselves. (Think jQuery.) Libraries are usually 'syntactic sugar' over something difficult.
+The term 'library' is often used interchangeably with the term 'framework', but they're actually a bit different. Libraries are _usually_ a little bit smaller, and generally serve the purpose of providing us with abstractions over complex code that we would otherwise have to write ourselves. (Think jQuery.) Libraries are sometimes 'syntactic sugar' over something difficult and may target specific functionality.
 
-Frameworks, on the other hand, offer us a bit more than just abstractions - they give us a lot more powerful ways to write our code, but at the same time, they prescribe a very strict and specific way for us to organize our code.
+Frameworks, on the other hand, offer us a bit more than just abstractions - they give us a lot more powerful ways to write our code, but at the same time, they prescribe a very strict and specific way for us to organize our code. Frameworks often try to provide **everything** required to develop a complete application.
 
-We'll see this demonstrated a bit better when we get into the code along, but React gives us a lot of cool features:
+According to the docs, **React is a library and an architecture.**   
+...wait, what???  
+You'll be writing entire applications in React in Mod 3 which can make it _feel_ more like a framework. You may even see it described in various places as a framework. It's important to note that you can add React components to an application, even if that entire application isn't using React.  In this way, React is more of a library - even if it's a **really** robust one.  
```

Comment:
Love this piece about using components in an app that otherwise doesn't use React. A great illustration of react being a library